# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bitfields/hotfix-install-t.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bitfield-0.2/Makefile.vanilla	2024-06-09 21:18:47.154000000 +0200
+++ bitfield-0.2/Makefile	2024-06-09 21:21:54.701000000 +0200
@@ -18,12 +18,12 @@
 	install -d $(DESTDIR)$(sysconfdir)/bash_completion.d
 	install -d $(DESTDIR)$(sharedir)/vim/addons/syntax
 	install -d $(DESTDIR)$(sysconfdir)/bitfield.d
-	install -m 755 -t $(DESTDIR)$(bindir) bitfield
+	install -m 755 bitfield $(DESTDIR)$(bindir)
 	install -m 644 -D bitfield-completions.sh \
 		$(DESTDIR)$(sysconfdir)/bash_completion.d/bitfield
-	install -m 644 -t $(DESTDIR)$(sharedir)/vim/addons/syntax bitfield.vim
-	install -m 644 -t $(DESTDIR)$(sysconfdir)/bitfield.d \
-		$(foreach f,$(configs),conf/$(f))
+	install -m 644 bitfield.vim $(DESTDIR)$(sharedir)/vim/addons/syntax
+	install -m 644 $(foreach f,$(configs),conf/$(f)) \
+		$(DESTDIR)$(sysconfdir)/bitfield.d \
 
 clean:
 
